interface/src/app/greenelab/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * This module includes a few utility components that will be shared in other
+ * modules to avoid duplicate codes.
+ */
+
+angular.module('greenelab.utils', [])
+.constant('ApiBasePath', '/api/v0/')
```

Comment:
I get that it's a constant, but not why it belongs in this file. My point is that `greenelab/utils.js` should not have a constant that is specific to accessing adage-server API endpoints.

It may very well be handy to have such a constant, but it should go somewhere else.